internal/services/nss/nss.go

```diff
@@ -205,17 +210,17 @@ func nssPasswdFromUsersPasswd(u users.UserEntry) *authd.PasswdEntry {
 }
 
 // nssGroupFromUsersGroup returns a GroupEntry from users.GroupEntry.
-func nssGroupFromUsersGroup(g users.GroupEntry) *authd.GroupEntry {
+func nssGroupFromUsersGroup(g types.GroupEntry) *authd.GroupEntry {
 	return &authd.GroupEntry{
 		Name:    g.Name,
-		Passwd:  "x",
+		Passwd:  g.Passwd,
```

Comment:
Worth being in a separate commit?

Reply:
It does belong to this commit, we only set the field now because we use it to store the identifier of a temporary group record. I added a comment to explain this.